raiden_contracts/tests/test_service_registry.py

```diff
@@ -51,7 +51,10 @@ def test_deposit_contract_without_service_registry_code(
     (A,) = get_accounts(1)
     call_and_transact(custom_token.functions.mint(100), {"from": A})
     depo = get_deposit_contract(
-        _token=custom_token.address, _release_at=UINT256_MAX, _withdrawer=A, _service_registry=A
+        _token=custom_token.address,
+        _release_at=UINT256_MAX,
+        _withdrawer=A,
+        _service_registry=A,
```

Comment:
Not related to this PR, but why are the arguments here prefixed with `_`?

Reply:
Those are the same parameter names used in the constructor of the `Deposit` solidity contract. They are chosen that way in the constructor to avoid name clashes with Deposit contract's attributes. Solidity doesn't have `self` or `this`, so this is the way we arrive at code like
```
token = _token
```
in the constructor to set the `token` attribute to the value given to the constructor.